cw_monero/pubspec.yaml

```diff
@@ -19,7 +19,10 @@ dependencies:
   flutter_mobx: ^2.0.6+1
   intl: ^0.18.0
   encrypt: ^5.0.1
-  polyseed: ^0.0.2
+  polyseed:
+    git:
+      url: https://github.com/cake-tech/polyseed_dart
+      ref: normalize-french-and-spanish
```

Comment:
will leave a comment here so we don't forget to update it back when the package is updated